server/src/config/database.config.ts

```diff
@@ -2,7 +2,7 @@ import { TypeOrmModuleOptions } from '@nestjs/typeorm';
 
 export const databaseConfig: TypeOrmModuleOptions = {
   type: 'postgres',
-  host: 'immich_postgres',
+  host: process.env.HOST || 'immich_postgres',
```

Comment:
```suggestion
host: process.env.DB_HOSTNAME|| 'immich_postgres',
```

Reply:
🤔  The weirdest thing happened. When I was going to make a PR I also noticed that `HOST` wouldn't be right, so I made another commit with the rename to `DB_HOST`, as you can see in the history just above `Rename database host env variable to DB_HOST (d55e7df)`. Then I went back and created a PR (with that commit included).   But for some reason the `Files changed` does not include that last commit 😕

Reply:
The force push did the trick 🎉 , the env is `DB_HOST`. We could change to `DB_HOSTNAME` if you want.

Reply:
Yes, let's change to `DB_HOSTNAME`